src/vs/workbench/contrib/terminalContrib/chat/browser/terminalChatWidget.ts

```diff
@@ -223,7 +223,7 @@ export class TerminalChatWidget extends Disposable {
 	}
 
 	private _resetPlaceholder() {
-		this.inlineChatWidget.placeholder = this._model.value?.welcomeMessage?.title ?? localize('askAI', 'Ask AI');
+		this.inlineChatWidget.placeholder = this._model.value?.welcomeMessage?.title ?? localize('askCopilot', 'Ask Copilot');
```

Comment:
Why is the welcome message missing anyway? We're trying to avoid hardcoding "copilot" terminology in the vscode repo

Reply:
I think that in panel chat this normally comes from the description of the default agent, and I'm not sure why the terminal has to have custom code for this anyway

Reply:
Yes the problem here is the welcome message isn't coming through which we need to investigate.